docs/codespell.txt

```diff
@@ -0,0 +1,4 @@
+requre
+noveau
+checkt
```

Comment:
Is the custom dictionary case-insensitive?

Reply:
Yes, it must be written with lower-case only

Reply:
Sad... `CheckT` and "I did checkt your work" are two different strings :)

Reply:
Lol inline-ignore does nothing and you get a new spelling error for the comment itself :))